src/luarocks/persist.lua

```diff
@@ -147,17 +144,11 @@ write_table = function(out, tbl, level, field_order)
       end
 
       write_value(out, v, level, sub_order)
-      if type(k) == "number" then
```

Comment:
it is redundant, but I wanted to store the pre-parsed info for speeding things up for `luarocks.loader` (though I don't remember if I measured it, and if I did, it was back in 2007 or so when I first wrote this code). The idea was that `luarocks.loader` should work as little as possible to have minimal impact on `require`.